Formula/aws-session-manager-plugin.rb

```diff
@@ -0,0 +1,18 @@
+class AwsSessionManagerPlugin < Formula
+  desc "Official Amazon AWS session manager plugin"
+  homepage "https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-working-with-install-plugin.html"
+  url "https://s3.amazonaws.com/session-manager-downloads/plugin/1.1.35.0/mac/sessionmanager-bundle.zip"
```

Comment:
Formula need to build from source

Reply:
This seems to be closed source.. Should it be moved to https://github.com/Homebrew/homebrew-cask ?

Reply:
In that case, yes

Reply:
Ok, I'll close this.